hack/app_sre_create_image_catalog.sh

```diff
@@ -102,7 +104,24 @@ RUN initializer --permissive
 CMD ["registry-server", "-t", "/tmp/terminate.log"]
 EOF
 
-docker build -f $DOCKERFILE_REGISTRY --tag "${REGISTRY_IMG}:${BRANCH_CHANNEL}-latest" .
+CATALOG_IMAGE="${REGISTRY_IMG}:${BRANCH_CHANNEL}-latest"
+docker build -f $DOCKERFILE_REGISTRY --tag "$CATALOG_IMAGE" .
+
+cleanup() {
+   docker rm -f catalog-image || true
+}
+
+trap cleanup EXIT
+cleanup
+docker run --name catalog-image -d --rm --network host "$CATALOG_IMAGE"
+sleep 10
```

Comment:
What are we doing here?

Reply:
Since the container is launched in the background there is no way to know when the grpc server is up. I could pool but to keep the script simple I just introduced a sleep. 10 seconds should be enough even on build machines.